--- origsrc/ffmpeg-1.2.4/configure	2013-10-02 18:36:22.000000000 -0500
+++ src/ffmpeg-1.2.4/configure	2013-12-05 02:17:48.520804000 -0600
@@ -3913,8 +3913,8 @@ enabled libnut     && require  libnut li
 enabled libopencore_amrnb  && require libopencore_amrnb opencore-amrnb/interf_dec.h Decoder_Interface_init -lopencore-amrnb
 enabled libopencore_amrwb  && require libopencore_amrwb opencore-amrwb/dec_if.h D_IF_init -lopencore-amrwb
 enabled libopencv  && require_pkg_config opencv opencv/cxcore.h cvCreateImageHeader
-enabled libopenjpeg && { check_lib openjpeg-1.5/openjpeg.h opj_version -lopenjpeg ||
-                         check_lib openjpeg.h opj_version -lopenjpeg ||
+enabled libopenjpeg && { check_lib2 openjpeg-1.5/openjpeg.h opj_version -lopenjpeg ||
+                         check_lib2 openjpeg.h opj_version -lopenjpeg ||
                          die "ERROR: libopenjpeg not found"; }
 enabled libopus    && require_pkg_config opus opus_multistream.h opus_multistream_decoder_create
 enabled libpulse && require_pkg_config libpulse-simple pulse/simple.h pa_simple_new
--- origsrc/ffmpeg-1.2.4/libavcodec/libopenjpegdec.c	2013-10-02 18:36:25.000000000 -0500
+++ src/ffmpeg-1.2.4/libavcodec/libopenjpegdec.c	2013-12-05 02:24:37.908289600 -0600
@@ -24,7 +24,9 @@
  * JPEG 2000 decoder using libopenjpeg
  */
 
+#if !defined(_WIN32) || defined(_WIN64)
 #define  OPJ_STATIC
+#endif
 
 #include "libavutil/common.h"
 #include "libavutil/intreadwrite.h"
--- origsrc/ffmpeg-1.2.4/libavcodec/libopenjpegenc.c	2013-10-02 18:36:25.000000000 -0500
+++ src/ffmpeg-1.2.4/libavcodec/libopenjpegenc.c	2013-12-05 02:24:34.475853800 -0600
@@ -24,7 +24,9 @@
  * JPEG 2000 encoder using libopenjpeg
  */
 
+#if !defined(_WIN32) || defined(_WIN64)
 #define  OPJ_STATIC
+#endif
 
 #include "libavutil/avassert.h"
 #include "libavutil/common.h"
